src/environment_windows.go

```diff
@@ -99,3 +103,93 @@ func (env *environment) getCachePath() string {
 	}
 	return env.homeDir()
 }
+
+//
+// Takes a registry path like "HKLM\Software\Microsoft\Windows NT\CurrentVersion" and a key under that path like "CurrentVersion" (or "" if the (Default) key is required).
+// Returns a bool and string:
+//
+//   true and the retrieved value formatted into a string if successful.
+//   false and the string will be the error
+//
+func (env *environment) getWindowsRegistryKeyValue(regPath string, regKey string) (string, error) {
+	env.trace(time.Now(), "getWindowsRegistryKeyValue", regPath, regKey)
```

Comment:
love that you also added this

Reply:
love me some perf metrics :)

Reply:
(same guy, different account, forgot I'm on the work laptop!)